dev/MRTCore/packaging/MrtCore.targets

```diff
@@ -4,6 +4,8 @@
   <!-- We want MRT Core to NOT be used in WAP or UWP projects. Instead, we want the in-box PRI generation tooling to be used. -->
   <PropertyGroup>
     <EnableCoreMrtTooling Condition="'$(EnableCoreMrtTooling)'=='' and '$(WindowsAppContainer)'!='true' and '$(MSBuildProjectExtension)' != '.wapproj'">true</EnableCoreMrtTooling>
+    <!-- If MSBuildRuntimeType is Core then we are using dotnet build because it will fail since ExpandPriContent is not available in the dotnet enviroment -->
```

Comment:
Do we know why "expandpricontent" isn't available there?

Reply:
Technical answer:
In DotNet Build,
MSBuildExtensionsPath = C:\Program Files\dotnet\sdk\9.0.201\
and Microsoft.Build.Packaging.Pri.Tasks.dll cannot be found there.

General answer:
I am not sure why. Maybe, Microsoft.Build.Packaging.Pri.Tasks.dll is just not provided by dotnet.

Reply:
This definitely smells: 
`    <AppxMSBuildToolsPath Condition="'$(AppxMSBuildToolsPath)' == ''">$(MSBuildExtensionsPath)\Microsoft\VisualStudio\v$(VisualStudioVersion)\AppxPackage\</AppxMSBuildToolsPath>`

If we can't find a way to load the entire packaging (PRI, MRT, Appx) environment from dotnet, without relying on potentially unavailable VS dlls, we may need to consider lifting that entire environment and eliminating the hybrid mixture.